src/snowflake/cli/plugins/snowpark/package/anaconda.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import logging
+from typing import List
+
+import requests
+from packaging.version import parse
+from snowflake.cli.plugins.snowpark.models import Requirement, SplitRequirements
+
+log = logging.getLogger(__name__)
+
+
+class AnacondaChannel:
+    url: str = "https://repo.anaconda.com/pkgs/snowflake/channeldata.json"
+
+    def __init__(self, packages):
+        self._packages = packages
+
+    def is_package_available(self, package: Requirement):
+        package_name = package.name.lower()
+        if package_name not in self._packages:
+            return False
+        if package.specs:
+            latest_ver = parse(self._packages[package_name]["version"])
+            return all([parse(spec[1]) <= latest_ver for spec in package.specs])
+        return True
+
+    @classmethod
+    def from_snowflake(cls):
+        response = requests.get(AnacondaChannel.url)
```

Comment:
Should we remove requests now, or it will be a different issue?

Reply:
I will do this later on, with checking also other place - I created issue for this